src/backend/oracle-data-api/src/main/resources/application.yml

```diff
@@ -101,9 +101,9 @@ cronjob:
 
 repository:
   # Defines the source of the lookup respository. Options are [csv, ords].
-  lookup: ${LOOKUP_REPOSITORY_SRC:csv}
+  lookup: ${LOOKUP_REPOSITORY_SRC:ords}
   # Defines the source of the dispute respository. Options are [h2, ords].
-  dispute: ${DISPUTE_REPOSITORY_SRC:h2}
+  dispute: ${DISPUTE_REPOSITORY_SRC:ords}
```

Comment:
I'm torn. Should the default for local development be H2 or ORDs. If we choose ORDs then it means that everyone will be tripping over each other and all using the same database, perhaps modifying eachother's records.

Reply:
I think switching to ORDS should be fine for dev environment since we will remove the H2 database eventually. If we need to debug something or modify majority of the data that could affect other work, then we can switch using H2 locally. Therefore, in my opinion, we can keep the default as ORDS for local development and switch when needed.